src/VisualStudio/Core/Impl/CodeModel/InternalElements/AbstractCodeMember.cs

```diff
@@ -152,7 +152,7 @@ public EnvDTE80.vsCMOverrideKind OverrideKind
             }
         }
 
-        internal virtual ImmutableArray<IParameterSymbol> GetParameters()
+        internal virtual ImmutableArray<SyntaxNode> GetParameters()
```

Comment:
@Pilchie `GetParameters` is only a valid call for the CodeElements `CodeProperty`, `CodeFunction` and `CodeDelegate`. If we make GetParameters return `ImmutableArray.CreateRange(CodeModelService.GetParameterNodes(LookupNode()));` in `AbtractCodeMember` then the other elements, like `CodeInterface` can potentially call GetParameters where the result will be an empty array which is different from throwing an exception.